main/feed_data.R

```diff
@@ -1,11 +1,17 @@
+library(plyr)
+library(readr)
+
 df <- data.frame(
   id = c(1,2,3,4,5),
   text = c("great","not great", "you jerk", "bananas", "orange jerks"),
   label = c(F, F, T, F, NA),
   training = c(T, T, T, T, F)
 )
+getwd()
+df <- read.csv("./src/data/both_t_data.csv")
+#df <- read.csv("~/Desktop/example_data.csv")
+df <- rename(df, c("X_id" = "id"))
```

Comment:
why not change it in the CSV?

Reply:
If CMU people pull data directly from MongoDB, they will use `_id` not `id` as the unique identifier, because `id` is used for something else. I didn't want to change my CMU CSV because just in case I need to update MongoDB with my result, I didn't want to mess up the column name.